aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 
+Open the **Add Website** window.
 
-Supported operating systems
------------------------------------
+	.. image:: pubIIS/_static/addwebsitecontextmenu.png
+	
+Configure the website.
 
-The following operations systems are supported:
+	.. image:: pubIIS/_static/addwebsite.png
+	
+In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
 
-- Windows 7 and newer
-- Windows 2008 R2 and newer
+	.. image:: pubIIS/_static/basicsettingscontextmenu.png
+	
+Set the **.NET CLR version** to **No Managed Code**.
+
+	.. image:: pubIIS/_static/editapppool.png
+
+Create a Data Protection Registry Hive
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Data Protection keys used by ASP.NET applications are stored in registry hives external to the applications. To persist the keys for a given application, you must create a registry hive for the application's application pool.
+
+For standalone IIS installations, you may use the `Data Protection Provision-AutoGenKeys.ps1 PowerShell script <https://github.com/aspnet/DataProtection/blob/dev/Provision-AutoGenKeys.ps1>`_ for each application pool used with an ASP.NET Core application. The keys will be persisted in the registry.
+
+In web farm scenarios, an application can be configured to use a UNC path to store its data protection key ring. By default, the data protection keys are not encrypted. You can deploy an x509 certificate to each machine to encrypt the key ring. See :ref:`Configuring Data Protection <data-protection-configuring>` for details.
+
+.. include:: ./dataProtectionWarning.txt
+
+Common Errors
+^^^^^^^^^^^^^
+
+The following is not a complete list of errors. Should you encounter an error not listed here, please leave a detailed error message in the DISQUS section below (click `Show comments` to open the DISQUS panel).
+
+To diagnose problems with IIS deployments, study browser output, examine the server's **Application** log through **Event Viewer**, and `enable logging <https://docs.asp.net/en/latest/fundamentals/logging.html>`_ in the application.
+
+A quick way to determine if the IIS reverse proxy to the Kestrel server is working properly is to perform a simple static file request (e.g., a stylesheet, script, or image) from the application's static assets (i.e., those in `wwwroot`) using `Static File Middleware <https://docs.asp.net/en/latest/fundamentals/static-files.html>`_. If the application can serve static files but MVC Views and other endpoints are failing, the problem is less likely related to IIS/module/Kestrel and more likely within the application itself (e.g., MVC routing, 500 Internal Server Error, etc.).
+
+In the following list, **ANCM Log** refers to the **ASP.NET Core Module** log and will be found on the path provided in the `stdoutLogFile` attribute of the `\<aspNetCore\>` element in `web.config`.
```

Comment:
I think it reads better to just inline the module name instead of using the ANCM acronym.